src/scroll-timeline-base.js

```diff
@@ -512,18 +512,20 @@ export class ScrollTimeline {
 }
 
 function getScrollParent(node) {
-  if (node === undefined)
+  if (!node)
     return undefined;
 
-  if (node.scrollHeight > node.clientHeight ||
-      node.scrollWidth > node.clientWidth) {
-    return node;
-  } else {
-    return getScrollParent(node.parentNode);
-  }
+  const style = getComputedStyle(container);
```

Comment:
I think you meant node here, not container.

Reply:
done.